src/js/components/Alert/Alert.tsx

```diff
@@ -35,7 +37,7 @@ const Alert: React.FC<Props> = ({
   return (
     <Tag {...other} className={classes}>
       {header && <h3 className="alert-heading mb-1">{header}</h3>}
-      <div className="d-flex align-items-start text-sm">
+      <div className={`d-flex text-sm w-100 ${containerFlexType}`}>
```

Comment:
I assume if this is undefined this shouldn't cause any trouble

Reply:
@nece2288 I think it should be alright, I've also given it a default value in case.